docs/pages/developer/use-api-key.en-US.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Using API Key with Zeabur
+ogImageTitle: Using API Key
+ogImageSubtitle: Using API Key with Zeabur
+---
+
+# Introduction
+
+You can interact with our GraphQL API / REST API by sending HTTP requests in any language. Before using it, you need to create an API Key in the [Dashboard](https://dash.zeabur.com).
+
+# Creating API Key
+
+In [Dashboard](https://dash.zeabur.com/), click on `Settings` in the dropdown menu on the right, then click on `Developer` in the sidebar on the left to see the developer options.
+
+![Developer Panel](/developer/api-key/developer.en-US.png)
+
+Find the `API Key` area, and click the `Generate New Token` button to generate an API Key.
+
+![Generate API Key](/developer/api-key/generate-api-key.en-US.png)
+
+Click the `View` button to view the API Key.
+
+![View API Key](/developer/api-key/view-api-key.en-US.png)
+
+Click the `Copy` button to copy the API Key.
+
+![Copy API Key](/developer/api-key/copy-api-key.en-US.png)
+
+# Using API Key
+
+## Authentication
+
+Zeabur API can optionally use the API Key for authentication. Ensure you have generated a valid API Key.
+
+All API requests should include your API Key in the `Authorization` HTTP header as shown below:
+
+```http
+Authorization: Bearer ZEABUR_API_KEY
+```
+
+# Issuing a REST API Request
```

Comment:
```suggestion
# Send a REST API Request
```

issue (v.): to [produce](https://dictionary.cambridge.org/zht/%E8%A9%9E%E5%85%B8/%E8%8B%B1%E8%AA%9E-%E6%BC%A2%E8%AA%9E-%E7%B9%81%E9%AB%94/produce) or [provide](https://dictionary.cambridge.org/zht/%E8%A9%9E%E5%85%B8/%E8%8B%B1%E8%AA%9E-%E6%BC%A2%E8%AA%9E-%E7%B9%81%E9%AB%94/provide) something [official](https://dictionary.cambridge.org/zht/%E8%A9%9E%E5%85%B8/%E8%8B%B1%E8%AA%9E-%E6%BC%A2%E8%AA%9E-%E7%B9%81%E9%AB%94/official).